test/npm/test-npm-test.js

```diff
@@ -64,30 +66,31 @@ test('npm-test: basic module failing', function (t) {
 test('npm-test: basic module no test script', function (t) {
   const context =
     makeContext.npmContext('omg-i-do-not-support-testing', sandbox);
-  npmTest(context, function (err) {
+  packageManagerTest('npm', context, function (err) {
     t.equals(err && err.message, 'Module does not support npm-test!');
     t.end();
   });
 });
 
 test('npm-test: no package.json', function (t) {
   const context = makeContext.npmContext('omg-i-dont-exist', sandbox);
-  npmTest(context, function (err) {
+  packageManagerTest('npm', context, function (err) {
     t.equals(err && err.message, 'Package.json Could not be found');
     t.end();
   });
 });
 
-test('npm-test: alternative test-path', function (t) {
+// Skipped since this does not work with shell scripts
+skip('npm-test: alternative test-path', function (t) {
```

Comment:
do we also need to skip for npm?

Reply:
yeah. I can differentiate yarn and npm here, but we try to find a `binary` named `npm` - the fact that `node` can run that binary is a weird coupling (IMO).

Can add it back in, though, if you want?

Note that it would probably work with a `yarn` not installed from brew as well